packages/python/src/components/PyDoc.tsx

```diff
@@ -4,239 +4,230 @@ import { ParameterDescriptor } from "../parameter-descriptor.js";
  */
-export function GoogleStyleDocParam(props: GoogleStyleDocParamProps) {
-  return (
-    <>
-      <GoogleStyleDocParamName name={props.name} />
-      <GoogleStyleDocParamType
-        type={props.type}
-        default={Boolean(props.default)}
-      />
-      <GoogleStyleDocParamDescription
-        children={props.children}
-        default={props.default}
+export function FunctionDoc(props: FunctionDocProps) {
+  const style = props.style ?? "google";
+  if (style === "google") {
+    return (
+      <GoogleStyleFunctionDoc
+        description={props.description}
+        parameters={props.parameters}
+        returns={props.returns}
+        yields={props.yields}
+        raises={props.raises}
+        note={props.note}
       />
-    </>
-  );
+    );
+  }
 }
 
-export interface GoogleStyleDocParamsProps {
-  parameters: ParameterDescriptor[] | string[];
+export interface ClassDocProps {
+  description: Children[];
+  parameters?: ParameterDescriptor[];
+  attributes?: GoogleStyleDocAttributeProps[];
+  note?: string;
+  style?: "google";
 }
 
 /**
- * A component that creates a GoogleStyleDoc block for parameters.
+ * A component that creates a ClassDoc block for classes.
  */
-export function GoogleStyleDocParams(props: GoogleStyleDocParamsProps) {
-  const parameters = normalizeParametersForDoc(props.parameters);
-  return (
-    <>
-      {"Args:"}
-      <Indent>
-        <List doubleHardline>
-          {parameters.map((param) => (
-            <GoogleStyleDocParam
-              name={param.name}
-              type={param.type}
-              default={param.default}
-            >
-              {param.doc}
-            </GoogleStyleDocParam>
-          ))}
-        </List>
-      </Indent>
-    </>
-  );
+export function ClassDoc(props: ClassDocProps) {
+  const style = props.style ?? "google";
+  if (style === "google") {
+    return (
+      <GoogleStyleClassDoc
+        description={props.description}
+        parameters={props.parameters}
+        attributes={props.attributes}
+        note={props.note}
+      />
+    );
+  }
 }
 
-export interface GoogleStyleDocReturnProps {
-  message: string;
+export interface ModuleDocProps {
+  description: Children[];
+  attributes?: GoogleStyleDocAttributeProps[];
+  todo?: string[];
+  style?: "google";
 }
 
 /**
- * A component that creates a GoogleStyleDoc block for parameters.
+ * A component that creates a ModuleDoc block for module-level documentation.
  */
-export function GoogleStyleDocReturn(props: GoogleStyleDocReturnProps) {
-  return (
-    <>
-      {"Returns:"}
-      <Indent>{props.message}</Indent>
-    </>
-  );
-}
-
-export interface GoogleStyleDocRaisesProps {
-  message: string;
+export function ModuleDoc(props: ModuleDocProps) {
+  const style = props.style ?? "google";
+  if (style === "google") {
+    return (
+      <GoogleStyleModuleDoc
+        description={props.description}
+        attributes={props.attributes}
+        todo={props.todo}
```

Comment:
Is this the only one that has `todo`?

Reply:
Revising https://google.github.io/styleguide/pyguide.html#docstrings, there's no explicit allowance or disallowance. Would you like to see it being added to all components?